components/ScheduleTable.tsx

```diff
@@ -0,0 +1,310 @@
+                        <th
+                          {
+                            ...column.getHeaderProps()
+                          }
+                          title={column.title}
+                          key={`${i}_${column.id}`}
+                          >
+                            {
+                              column.render('Header')
+                            }
+                          </th>
+                      )
+                    )
+                  }
+                </tr>
+              ))
+            }
+          </TableHeader>
+          <TableBody {...getTableBodyProps()}>
+            {
+              page.map((row, i) => {
+                prepareRow(row);
+
+                return (
+                  <tr {...row.getRowProps()} key={i}>
+                    {
+                      row.cells.map((cell, i) => {
+                        return (
+                          <td
+                            {...cell.getCellProps()}
+                            key={i}
+                          >
+                            {
+                              cell.render('Cell')
+                            }
+                          </td>
+                        )
+                      })
+                    }
+                  </tr>
+                )
+              })
+            }
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </SkeletonTheme>
+  )
+}
+
+const TableContainer = styled.div`
+  border: 1px solid ${({ theme }) => theme.colors.grey500};
+  border-top: none;
+  overflow-x: auto;
+  overflow-y: hidden;
+  border-radius: 10px;
+
+  tr:not(:last-child) th,
+  tr:not(:last-child) td {
+    border-bottom: 1px solid ${({ theme }) => theme.colors.grey500};
+  }
+`;
+
+const Table = styled.table`
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 100%;
+
+  border-radius: inherit;
+`;
+
+const TableHeader = styled.thead`
+  background-color: ${({ theme }) => theme.colors.grey900};
+  color: ${({ theme }) => theme.colors.grey100};
+  position: relative;
+
+  tr {
+    display: table-row;
+  }
+
+  th {
+    height: 50px;
+    font-weight: 400;
+    background-color: ${({ theme }) => theme.colors.grey900};
+    position: relative;
+    text-align: left;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+`;
+
+const TableBody = styled.tbody`
+  background-color: ${({ theme }) => theme.colors.grey100};
+  margin: 0 auto;
+  display: table-row-group;
+  vertical-align: middle;
+  position: relative;
+
+  th {
+    display: table-cell;
+    text-align: left;
+    font-weight: 400;
+    background-color: ${({ theme }) => theme.colors.grey200};
+    position: sticky;
+    left: 0px;
+  }
+
+  td {
+    font-family: Montserrat, Arial, Helvetica, sans-serif;
+    padding: 10px;
+    text-align: left;
+  }
+
+  tr {
+    &:hover {
+      background-color: rgba(1, 131, 218, 0.1);
+    }
+  }
+`;
+
+const Pagination = styled.div`
```

Comment:
I'm good at implementing designs but my personal creations are always the best. Let me know if you have any better ideas on how to make the pagination elements look.